src/main/java/frc/robot/RobotContainer.java

```diff
@@ -54,14 +58,8 @@ private void configureButtonBindings() {
     // Back button zeros the gyroscope
     new Button(m_controller::getBackButton)
             // No requirements because we don't need to interrupt anything
-            .whenPressed(m_GyroSubsystem::zeroBestGyro)
-            .whenReleased(m_GyroSubsystem::setIsZeroingFalse);
-
-    
-    //new Button(m_controller::getAButton).whenPressed(new climberActuatorIn(servoSubsystem));
-    
-    //new Button(m_controller::getBButton).whenPressed(new climberActuatorOut(servoSubsystem));
-  
+            .whenPressed(m_drivetrainSubsystem::resetFieldOriented)
```

Comment:
make sure to bind setIsZeroingFalse to when released otherwise the heading control wont reactivate